pkgs/applications/science/electronics/openboardview/default.nix

```
@@ -0,0 +1,45 @@
+{ stdenv, lib, fetchFromGitHub, gitUpdater
+, cmake, pkg-config, python3, SDL2, fontconfig, gtk3 }:
+
+stdenv.mkDerivation rec {
+  pname = "openboardview";
+  version = "9.0.3";
+
+  src = fetchFromGitHub {
+    owner = "OpenBoardView";
+    repo = "OpenBoardView";
+    rev = "${version}";
+    sha256 = "sha256-0vxWFNM9KQ5zs+VDDV3mVMfHZau4pgNxQ1HhH2vktCM=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ cmake pkg-config python3 ];
+  buildInputs = [
+    SDL2 fontconfig gtk3
+  ];
+
+  postPatch = ''
+    substituteInPlace src/openboardview/CMakeLists.txt \
+      --replace "SDL2::SDL2main" ""
+  '';
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
+    "-DGLAD_REPRODUCIBLE=On"
+  ];
+
+  passthru = {
+    updateScript = gitUpdater {
+      inherit pname version;
+      ignoredVersions = ''.*\.90\..*'';
+    };
+  };
+
+  meta = with lib; {
+    description = "Linux SDL/ImGui edition software for viewing .brd files";
+    homepage = "https://github.com/OpenBoardView/OpenBoardView";
+    license = licenses.mit;
+    platforms = [ "x86_64-linux" ];
```

Are you sure this won't work for AArch64?
```suggestion
    platforms = platforms.linux;
```

---

I agree, this was an oversight. It should work fine on any Linux. It may even work on Darwin but I wouldn't enable that yet until it is tested as it may require additional inputs.